crates/sui/src/client_commands.rs

```diff
@@ -775,32 +773,21 @@ impl SuiClientCommands {
 
                 let package = compile_package_simple(build_config, package_path)?;
                 let modules: Vec<_> = package.get_modules().cloned().collect();
-                let mut metered_verifier_config =
```

Comment:
@oxade could you please review this part?
You'll notice I defined an entry point in the execution traits that deals with this specifically and I am not sure how critical. I think it'll be fine to keep that around so no big deal, but I wonder if you (or anybody else) think there is a good way to do this